web/garbage.go

```
@@ -25,19 +28,23 @@
 )
 
 // getRandomData 生成指定大小的随机数据块
-func getRandomData(length int) []byte {
-	data := make([]byte, length)
-	_, err := rand.Read(data) // 使用 crypto/rand.Read 获取随机数据，返回读取的字节数和 error
+func getRandomData(size int) ([]byte, error) {
+	data := make([]byte, size)
+	_, err := rand.Read(data) // 使用 crypto/rand.Read
```

golangci-lint (1.64.8)

SA1019: rand.Read has been deprecated since Go 1.20 because it shouldn't be used: For almost all use cases, [crypto/rand.Read] is more appropriate. If a deterministic source is required, use [math/rand/v2.ChaCha8.Read]. 

(staticcheck)